src/main/java/com/powsybl/openloadflow/network/DiscreteVoltageControl.java

```diff
@@ -56,6 +56,11 @@ public void addController(LfBranch controllerBranch) {
         controllers.add(controllerBranch);
     }
 
+    public boolean removeController(LfBranch controllerBranch) {
```

Comment:
You don't need this function in that PR as you will have to merge voltage controls and so totally deleted one.